addons/mail/static/src/discuss/call/common/call_settings.xml

```diff
@@ -2,13 +2,15 @@
 <templates xml:space="preserve">
 
     <t t-name="discuss.CallSettings">
-        <t t-call="discuss.CallSettings.content"/>
+        <ActionPanel t-if="props.withActionPanel" title.translate="Voice settings" resizable="!env.config">
```

Comment:
`env.config` is quite cryptic to tell this is discuss client action (for public page).
I would define a `createSubEnv({ inDiscussClientAction: true })` in `discuss_client_action`, so that the check is on `env.inDiscussClientAction`.

Reply:
I realized I don't need the resizable as false anymore